src/genie/libs/parser/iosxr/tests/ShowRunIpslaOperation/cli/equal/golden_output_2_expected.py

```diff
@@ -0,0 +1,48 @@
+expected_output = {
+  "ipsla": {
+    "operations": {
+      "operation_ids": [
+        {
+          "oper_id": 100,
+          "oper_types": {
+            "type": {
+              "name": "udp jitter",
+              "vrf": "VRF-1",
```

Comment:
You can simply like below and no need to use `ListOf()` as long as it can be distinguished by operation id. I believe operation id is unique.
```suggestion
    "operation": {
      100: {
        "type": {
          "udp jitter": { 
            "vrf": "VRF-1",
```

Reply:
I committed this and reverted as this may break the way I wanted to keep the operation ids, I like to have list of them rather than dictionary, also the operations ket may have other parameters too. Please review again

Reply:
why `list`? as I said it works even with dict key/value and no need to have list there. we usually don't have list in parsed dictionary because we will need to do looping for elements until we find it. but with dict key/value, as long as we know the operation id, we can pick up the key/values relating to hte operation id quickly. so I'm seeing benefit with key/value instead of list here.

Reply:
hmm OK, make sense, let me change that to the dict, probably need to update the script and schema both.

Reply:
thanks for your understanding. I will be waiting for the update!

Reply:
Just updated the script and schema, please review and let me know if this looks good

Reply:
confirmed the change.